code/config.dat

```diff
@@ -1,4 +1,4 @@
-root,C:\Users\yq904481\research\repos\HUXt
+root,D:\Dropbox\python_repos\HUXt\
```

Comment:
We should probably find a better way to do this than asking people to change the root directory. You might want to add this file into your .gitignore file so this change doesn't get pushed.